ShestakUI/Modules/Chat/ChatFrames.lua

```diff
@@ -117,7 +117,7 @@ local function SetChatStyle(frame)
 	if T.Cata then
 		_G[format("ChatFrame%sMinimizeButton", id)]:Kill()
 	else
-		_G[format("ChatFrame%sButtonFrameMinimizeButton", id)]:Kill()
+		-- _G[format("ChatFrame%sButtonFrameMinimizeButton", id)]:Kill() -- FIXME
```

Comment:
I was able to get this working using the same as whats setup for Cata.

```suggestion
		-- _G[format("ChatFrame%sMinimizeButton", id)]:Kill()
```